src/resources/languages.h

```diff
@@ -10,6 +10,33 @@
 int lang_init(void);
 void lang_close(void);
 
+/*! \brief OMF 2097 String ID
+ *
+ * These string IDs match OMFv2.1 (Epic Challenge Arena)
+ */
+enum
+{
+    // there are 10 HARs
+    LANG_STR_HAR = 31,
+    LANG_STR_NEWSROOM_NEWCHAMPION = 79,
+    // there are 3*2 pronouns
+    LANG_STR_PRONOUN = 81,
+    // there are 24*2 newsroom texts
+    LANG_STR_NEWSROOM_TEXT = 87,
+
+    LANG_STR_COUNT = 1013,
```

Comment:
Could the 990 old thing be here too, or would it be too confusing?

Reply:
I avoided doing so, as it's not from the same sequence as the rest of the `LANG_STR` enum

Reply:
Right, well, in the place where the 990 is used it's not that clear where it comes from, maybe it needs some comment.

Reply:
Moved it to a named constant in languages.c with a comment